gradle/spotbugs.gradle

```diff
@@ -7,6 +7,26 @@ allprojects {
     omitVisitors = ["FindDeadLocalStores"]
     effort = "max"
     excludeFilter = file("$rootDir/gradle/spotbugs-exclude.xml")
+    toolVersion = '4.1.3'
```

Comment:
Why older version here?

Reply:
This is the latest spotbugs _tool_ version, the gradle plugin has a different version cadence.

Reply:
And do you _have_ to specify it here? If you don't will anything break?

Reply:
no, nothing breaks; if left unspecified it will revert to whatever version the plugin defines. IMO it is probably better to peg the tool version instead of leaving it undefined.

Reply:
Extra unnecessary configuration is extra code to maintain. If we already peg the version of the plugin, it should enough, no?

Reply:
If you upgrade the gradle plugin, you get additional plugin fixes and/or configuration but you _might_ also get a new version of the _spotBugs_ tool. If you're explicit about the tool version, you can be reasonably confident of it not changing underneath your feet - having said that I don't feel strongly about it, so I'm happy to go with whatever direction you decide.

Reply:
I prefer not to fix tool versions unless there is an actual reason for that :) Upgrade all the things! :)